ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v1/container/clientlibs/editor/js/editDialog.js

```diff
@@ -489,6 +493,22 @@
         }
     });
 
+    function registerRestEndpointUrlValidator() {
+        $(window).adaptTo("foundation-registry").register("foundation.validation.validator", {
+            selector: "[data-rest-endpoint-url-validation]",
+            validate: (el) => {
+                const url = el.value;
+                // Regex to validate absolute URLs starting with http:// or https:// only
+                const absoluteUrlPattern = /^https?:\/\/.+$/i;
+                if (!absoluteUrlPattern.test(url)) {
+                    return Granite.I18n.getMessage(
+                        "Enter a valid URL for the POST request. Ensure the URL starts with 'https://' or 'http://' and follows this structure: https://www.server.com/path/to/resource."
```

Comment:
@pavi41 relative URL is also a valid URL - so we need to change the text.
It should be a simple message - "Please enter the absolute path of the REST endpoint."

Also, we need to conditionally update the dialog to update the help text.